examples/flutter_gallery/lib/demo/fortnightly/fortnightly.dart

```diff
@@ -80,7 +79,8 @@ class FruitPage extends StatelessWidget {
  of water to make that much pomegranate juice. Water the Resnicks get from their
  majority stake in the Kern Water Bank. How did one family come to hold control
  over water meant for the whole central valley of California? The story will shock you.
-'''.replaceAll('\n', '');
+'''
+      .replaceAll('\n', '');
```

Comment:
not clear that this is in an improvement

Reply:
Agree. Will file an issue on _dart_style_.

Reply:
Filed https://github.com/dart-lang/dart_style/issues/796